tests/ft_erlang.vader

```diff
@@ -0,0 +1,127 @@
+Include: include/setup.vader
+
+Execute (erlc: common settings):
+  new
+  file src/myapp.erl
+
+  let args = neomake#makers#ft#erlang#GlobPaths()
+  AssertEqual args[:-2], ['-pa', 'ebin', '-I', 'include', '-I', 'src', '-o']
+  let output_dir = args[-1]
+  Assert isdirectory(output_dir)
+  bwipe
+
+Execute (erlc: output to _build/neomake if _build is present):
+  let root = tempname()
+  call mkdir(root, 'p')
+  let s:old_cwd = fnamemodify('.', ':p')
```

Comment:
You can use `getcwd()` to get the current dir.

And then it would be good to wrap it with `try … finally` to `cd` back - so that a failing test restores the cwd.